test/integration_test.py

```diff
@@ -162,7 +165,8 @@ class IndexingIntegrationTest(IntegrationTestCase, AlwaysTearDownTestCase):
 
     def setUp(self) -> None:
         super().setUp()
-        self.pruning_seed = random.randint(0, sys.maxsize)
+        self.pruning_seed = randint(0, sys.maxsize)
+        self.random = Random(x=self.pruning_seed)
```

Comment:
Why parametrize `Random` with `x`?

Reply:
It's a badly named parameter imo but `x` is the seed used by the `Random` instance. I'm not sure what we gain by passing it as a keyword arg but that's how `Random` is instantiated for the IT on `develop`:
https://github.com/DataBiosphere/azul/pull/3460/commits/1549a8ce0d4b2f68a05263365a8bb596b480ca9a#diff-4e904e078d9dedfc711df509920616607e93d4970e8a1115465fabc13bca0e00L654

Reply:
Ah excellent, precedent is an authoritative source, thanks for the reference.

Reply:
I think `x` was used erroneously because the typing stub calls it that. The [docs](https://docs.python.org/3.8/library/random.html#random.Random) define this as an optional positional argument so the stub should probably have been `__init__(seed=None, /)`

Anyways, since this PR removes the code establishing precedent, we can easily fix this now.